wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/DeveloperToolsView.swift

```diff
@@ -97,9 +97,14 @@ private struct TextItemCell: View {
                 .lineLimit(1)
                 .truncationMode(.middle)
                 .foregroundColor(.secondary)
+                .contextMenu {
+                    SwiftUI.Button(
+                        action: { UIPasteboard.general.string = value },
+                        label: { Text("Copy") }
+                    )
```

Comment:
As discussed in another channel, this duplicates logic we have to copy on tapping the cell, however this provides better UI hints. However, this view shouldn't have any business logic in it, i.e I don't think we should be modifying the pasteboard here in the view.